include/podio/Frame.h

```diff
@@ -247,8 +246,8 @@ class Frame {
   ///
   /// @param key   The name under which this parameter should be stored
   /// @param value The value of the parameter. A copy will be put into the Frame
-  inline void putParameter(const std::string& key, const char* value) {
-    putParameter<std::string>(key, value);
+  inline void putParameter(const std::string& key, std::string value) {
+    putParameter<std::string>(key, std::move(value));
```

Comment:
Just having the `std::string` and `const char*` overloads are not enough for catching all implicit conversions?

Reply:
Ah right. Overload resolution is fun ;)